src/apiProxy/apiConfigCreators.js

```diff
@@ -71,6 +71,16 @@ export function event(params) {
 }
 
 /**
+ * admin endpoints
+ */
+export function admin(params) {
+	return {
+		endpoint,
```

Comment:
`endpoint` is undefined here.

The current version of the platform allows you specify the `endpoint` directly in your query - no more need for changes to `apiConfigCreators`, which is now deprecated